automated_testing/test/constants/submissions-details-page-selectors.ts

```diff
@@ -0,0 +1,3 @@
+const selectors = {
```

Comment:
Is this empty file neccessary?

Reply:
everything in submission-details was created in order to help Simona with the set up for her tests. 
I'm going to write in her branch and it'll not stay empty. If I have to, I will delete it here.